Parse/cloud/main.js

```diff
@@ -355,6 +364,59 @@ Parse.Cloud.define("postVent", async (request) => {
   return;
 });
 
+Parse.Cloud.define("createGroup", async (request) => {
+  const Group = Parse.Object.extend("GroupDetails");
+  const group = new Group();
+  const authorId = request.params.authorId;
+  var groupMembershipIds = request.params.groupMembershipIds;
+  var groupName = request.params.groupName;
+
+  const userQuery = new Parse.Query("User");
```

Comment:
One very important thing I would like to see here would be input data validation. For example, what happens if the input params are incorrect/invalid, like, there's an `undefined` groupName, or undefined authorId? 

In these cases you can handle it like this

```
if (authorId === undefined || groupName === undefined) {
  throw Error("Missing author or group name!");
}

// now below, we know we have good data and can use it
```